packages/amplify_datastore_plugin_interface/test/testData/ModelProvider.dart

```diff
@@ -27,14 +28,9 @@ export 'PostAuthComplex.dart';
 
 class ModelProvider implements ModelProviderInterface {
   @override
-  String version = "6319a5b46c3ea5e5817b63396bc464bb";
+  String version = "33b1717ef9afb390e3f7431e8f3859fc";
   @override
-  List<ModelSchema> modelSchemas = [
-    Blog.schema,
-    Comment.schema,
-    Post.schema,
-    PostAuthComplex.schema
```

Comment:
question: Why is `PostAuthComplex` omitted now?

Reply:
good call.  it shouldn't ... added it back in